src/com/gui/Configuration.java

```diff
@@ -0,0 +1,119 @@
+package com.gui;
+
+import com.utils.writers.PropertiesWriter;
+
+import java.util.Scanner;
+
+/**
+ * This class represent the Configuration console interface
+ */
+public class Configuration extends Console {
+    private PropertiesWriter propertiesWriter = new PropertiesWriter();
+
+    public void start() {
+        Boolean exit = false;
+        do {
+            display("------Configuration Settings------" + "\n");
+            display("1.Output type");
+            display("2.Level");
+            display("3.Algorithm");
+            display("4.Interface");
+            display("5.Exit");
+            display("Select an option:");
+            input = new Scanner(System.in);
+            int option = input.nextInt();
+            switch (option) {
+                case 1:
+                    enterOutputType();
```

Comment:
I am not huge fan of having switches, instead of that try to use polymorphism or try to use the open closed principle of SOLID or reflection with a hash map